### ISSUE TITLE
t!at Day is off by 1

### ISSUE THREAD

**Issue by midblue** [midblue/timezone-bot#36]
from #suggestions: "when I run "t/at 9pm san francisco", it says "Tue 9:00PM" at the top but show Monday 9pm in the list below"

**Comment by midblue** [comment]
![Screenshot_20201213-102811_Discord](https://user-images.githubusercontent.com/11934922/102018046-01726000-3d6b-11eb-81b7-8e3921556f88.jpg)

**Comment by midblue** [comment]
Also,

```
DrTim2Today at 8:27 AM
t!at Sat 2PM GMT
TimezoneBot
BOT
Today at 8:27 AM
At Sat 2:00 PM in Gmt, it will be...
☀️Sat 08:00 AM - Central Time
☀️Sat 09:00 AM - Eastern Time
☀️Sat 11:00 AM - Chile Time
☀️Sat 02:00 PM - British Time
☀️Sat 03:00 PM - Central European Time
☀️Sat 04:00 PM - Eastern European Time
☀️Sat 05:00 PM - Arabian Time
🌙Sat 07:30 PM - India Time
🌙Sun 12:00 AM - Australian Eastern Time
☀️Sun 06:00 AM - Pacific Time
```

**Comment by bouncingmolar** [comment]
I think this is the same problem as #35

**Comment by midblue** [comment]
It seems that the server is using the day that it is in (GMT) as the basis for day calculations. Fixable but a little tricky.

**Comment by zsawyer** [comment]
From what I can tell this issues was fixed with commit 034ad72a380058c7423d5ae427509cf98a0eb90e
I cannot reproduce this issue. 

![image](https://user-images.githubusercontent.com/897208/130529940-a2a7d125-2718-45a8-bff5-699896323b25.png)
